payments/card/direct.md

```diff
@@ -66,92 +73,20 @@ An example of an expanded `POST` request is available in the
 
 {% include alert-callback-url.md payment_instrument="card" %}
 
-## Step 2a: Create an authorization transaction
+{% include alert.html type="neutral" icon="report_problem"
+body="**Step 2** is to create an authorization transaction. Implement only
+Step 2a if **3-D secure authentication is enabled**. Note that if the issuer
+does **not require** 3-D Secure authentication, implement only Step 2b.
+You will see that the requests are the same for both steps.
+The significant difference is in the response headers,
```

Comment:
The headers are the same, are they not? It's the operations that change.
If the headers are suddenly significant we have to document these.

Reply:
good point!